spec/serializers/yellow_ribbon_program_source_serializer_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe YellowRibbonProgramSourceSerializer, type: :serializer do
+  subject { serialize(yellow_ribbon_program_source, serializer_class: described_class) }
+
+  let(:yellow_ribbon_program_source) { create :yellow_ribbon_program_source }
+  let(:data) { JSON.parse(subject)['data'] }
+  let(:attributes) { data['attributes'] }
+
+  it 'includes city' do
+    expect(attributes['city']).to eq(yellow_ribbon_program_source.city)
+  end
+
+  it 'includes contribution_amount' do
+    expect(attributes['contribution_amount']).to eq(yellow_ribbon_program_source.contribution_amount)
```

Comment:
This test is currently failing as `contribution_amount` is a string instead of (what it should be) an integer.